.github/workflows/release.yaml

```diff
@@ -15,7 +15,7 @@ permissions:
 
 jobs: 
   release:
-    runs-on: ["self-hosted", "1ES.Pool=app-routing-operator-github-runner-pool"]
+    runs-on: ["self-hosted", "1ES.Pool=${{ vars.RUNNER_BASE_NAME }}-ubuntu"]
```

Comment:
📓 💡 Post this I think next will be web hook PR for the MCR hook, this looks good, thanks.